test/passing/tests/doc_comments.mli.ref

```diff
@@ -78,6 +78,8 @@ val k : k
     @author foo
     @version foo
     @see <foo> foo
+    @see <https://slash-create.js.org/#/docs/main/latest/class/SlashCreator?scrollTo=registerCommandsIn>
+    this url is very long
```

Comment:
This text is part of the `@see` tag and it's not obvious from the indentation.
Should we indent tag arguments that wrap ?

Reply:
I think we should indeed, I've added an indent of 2 cols